### PR TITLE
Add label visualization in frontend

### PR DESCRIPTION
**Pull request by jteijema**
It also changes the interactive dashboard a little, to more clearly see the scatterplot!

### CODE REVIEW

**Comment by jteijema** [comment]
Apparently, scope creep is real, and it is here.

This PR will:
Add labels to front end
Add support for benchmark datasets
Remove the --testfile function
Remove `clustering.py` and `dim_reduct` as this was legacy code
Give a makeover to the frontend
Use `load_data` instead of `from_file` to create an ASReview data object
Rename main function to adhere to pep8
Use batch_encode instead of looping over df
Remove old visualization code

That's all.
@J535D165

File: asreviewcontrib/semantic_clustering/semantic_clustering.py [hunk 1]
```diff
@@ -2,39 +2,52 @@
...
+
+    # remove emptry abstracts
+    data = data[data['abstract'] != '']
+
+    if REMOVE_DUPLICATES:
+        try:
+            data["dup"] = asreview_data_object.df["duplicate_record_id"]
+            print("Size before removing duplicates is {0}".format(len(data)))
+            data = data[data['dup'].isna()]
+            print("size after removing duplicates is {0}".format(len(data)))
```
Comment (J535D165) [inline asreviewcontrib/semantic_clustering/semantic_clustering.py]:
```suggestion
            print("Size after removing duplicates is {0}".format(len(data)))
```

**Review by J535D165 — APPROVED** [review]
Happy to see all the improvements :)

**Comment by J535D165** [comment]
Nice!

**Pull request merged**